### ISSUE TITLE
Wildlife Spotter template

### ISSUE THREAD

**Issue by RhiannonStephens** [zaxen/volunteer-portal#3]
Filmstrip not right
![image](https://user-images.githubusercontent.com/10238220/32637179-7a02c354-c60c-11e7-839f-8b77f7720625.png)

**Comment by RhiannonStephens** [comment]
Can we change this sentence:
![image](https://user-images.githubusercontent.com/10238220/32637207-a1379d82-c60c-11e7-9552-163631fc8b9e.png)

To be:
Only classify the image highlighted with a black border. Images on either side are used to help you classify this image. 

Can we make sure the middle image is always black and that if you scroll over the others then they turn orange??

@zaxen can you make the middle always stay black. It currently changes to orange and the other thumbnails have a black border.

![image](https://user-images.githubusercontent.com/10238220/32818692-dd995800-ca18-11e7-8292-443f700aafd9.png)
 Sometimes the filmstrip is not working, other times it is??

**Comment by RhiannonStephens** [comment]
I deleted some images but their boxes still remain 
![image](https://user-images.githubusercontent.com/10238220/32637311-3317afee-c60d-11e7-91b7-0ea13c07e427.png)


This doesn't happen if you load by csv

**Comment by RhiannonStephens** [comment]
Drop down lists need to be the same as the Type drop down
 
![image](https://user-images.githubusercontent.com/10238220/32637357-784e5d24-c60d-11e7-8836-19f3485faa73.png)
![image](https://user-images.githubusercontent.com/10238220/32637364-7e871dde-c60d-11e7-8cd7-be8be2e812b3.png)

**Comment by RhiannonStephens** [comment]
Expedition list is showing Wombat chosen but when you go into the task it isn't showing that anything has been 
![image](https://user-images.githubusercontent.com/10238220/32637706-83576452-c60f-11e7-8477-14d4cbf90941.png)

![image](https://user-images.githubusercontent.com/10238220/32637690-75d9ecfa-c60f-11e7-9f6b-7227fc7d2f6b.png)

**Comment by RhiannonStephens** [comment]
I added the Raven from a csv
When I go in to it to edit and add photos the + and upload buttons will not work
![image](https://user-images.githubusercontent.com/10238220/32639092-463be064-c616-11e7-8189-e953e1d76ee1.png)
 I then deleted the Raven and it will not go from template

**Comment by RhiannonStephens** [comment]
Long description keeps disappearing
![image](https://user-images.githubusercontent.com/10238220/32639730-50bca2be-c619-11e7-983c-a1f15d404fe2.png)

**Comment by zaxen** [comment]
@RhiannonStephens these should be done now - except the entries not disappearing - I suspect that one is caused by not removing the actual animal entry and just blanking out the names, descriptions, images fields.  I'll add a filter that removes all animals with no names, descriptions or images when you save in the WS template admin UI.

**Comment by RhiannonStephens** [comment]
@zaxen does this build incorporate the new build from dev?

**Comment by zaxen** [comment]
@RhiannonStephens yes, this is based off the ALA develop branch (which is running on volunteer-dev.ala.org.au)

**Comment by sbearcsiro** [comment]
@RhiannonStephens I've tweaked the film strip so that the current image always has the black border and the selected image is orange.  The sequence images selection always seems to work for me - what issues are you seeing?

**Comment by RhiannonStephens** [comment]
@zaxen I've asked around about the white dots that make it look like a filmstrip and the general consensus is to remove them and just have it grey, more like the old way
![image](https://user-images.githubusercontent.com/10238220/33697769-b6a83afe-db5c-11e7-82e0-77b761d02f24.png)


And are you able to put the words 'Only classify' and 'black' in bold in the sentence:
Only classify the image highlighted with a black border. Images on either side are used to help you classify this image.

**Comment by RhiannonStephens** [comment]
Wildlife spotter banner image - I have the image which i have attached. Are you able to make it fit?
![ws](https://user-images.githubusercontent.com/10238220/33698071-4e8d84b8-db5e-11e7-8669-a47b10c56c31.jpg)

**Comment by zaxen** [comment]
@RhiannonStephens I've tweaked the banner so that it's no longer fixed for wildlife spotter (ie it will scroll with the page, rather than the page scroll over it.  Making it fit nicely and have it fixed is basically impossible.  I've also made the default WS banner the image you provided.

I've also removed the dots from the filmstrip and made the help text bold.  Are we good to send this to the ALA for merging now?
